### DISCUSSION TITLE
Add LaTeX support

### DISCUSSION THREAD

**Discussion by eurunuela** [roam-garden/i#14] [Feature requests & Ideas] [answered]
The title says it all. My notes contain a lot of equations so this is kind of an essential feature for me.

**Comment by Stvad** [comment, marked as answer]
Should be reasonably simple - https://www.gatsbyjs.com/plugins/gatsby-remark-katex/ supplies a compatible implementation, need to make sure that Roam flavor matches what that plugin provides.  

A related discussion https://github.com/mathieudutour/gatsby-digital-garden/issues/41

Added to the Roadmap: https://github.com/roam-garden/i/projects/1#card-56649817

**Reply by Stvad** [reply]
Important note is not to forget to disable throwing errors on parsing, so those won't break the build https://katex.org/docs/options.html

**Reply by Stvad** [reply]
![image](https://user-images.githubusercontent.com/81347/111859612-5e982780-88ff-11eb-9c13-bd41244ed839.png)
shipped

**Reply by Stvad** [reply]
implementation has some peculiarities because of https://github.com/remarkjs/remark-math/issues/57
